packages/analytics/addon/components/ca-report-preview.hbs

```diff
@@ -33,6 +33,7 @@
               <td
                 data-t={{this.getXLSXType entry.value}}
                 data-v={{this.cleanValue entry.value}}
+                class={{this.getCSSClass entry.value}}
```

Comment:
Nitpick:
I would prefer if this were just a helper since it doesn't require any context.
```suggestion
                class={{optional-multiline entry.value}}
```

Reply:
This already is a helper function
https://guides.emberjs.com/v4.12.0/components/helper-functions/#toc_local-helper-functions

I just think calling it with this makes it clear its a local helper